### ISSUE TITLE
MovieClip timescale

### ISSUE THREAD

**Issue by Ch1pStar** [jiborobot/pixi-animate#47]
Is this functionality supported at the moment, I didn't find anything that would indicate so in the code?
If no, do you have any plans for adding it in the future?
I'm currently working on a project that would almost certainly require this and I imagine it would be useful for others as well.

**Comment by bigtimebuddy** [comment]
MovieClips have a framerate which is independent (but defaults to the root stage or parent's framerate upon construction). This can be changed to adjust the playback rate of the clip:
http://jiborobot.github.io/pixi-animate/PIXI.animate.MovieClip.html#framerate

**Comment by Ch1pStar** [comment]
@bigtimebuddy Thanks, I totally overlooked the framerate property! A small issue is having to set the framerate of each movie clip separately(I can't set a framerate for the entire stage), but its not a problem for me right now.

One difference with the TweenMax timescale though, is there is no interpolation between key frames. Say an animation is created with 60fps and we set the animation to 30fps, the empty frames could interpolate between the previous and next key frames so that the animation is not choppy.

**Comment by bigtimebuddy** [comment]
That is correct. PixiAnimate exports discrete frames of animation so loss-less interpolation (like with TweenMax) is not possible.
